### DISCUSSION TITLE
知识库 QA分段模式建议使用 GPT-4o 去跑，不然成本太高了

### DISCUSSION THREAD

**Discussion by KylinShinlyLin** [langgenius/dify#5624] [Help]
It is recommended to use GPT-4o to run the knowledge base QA segmentation mode, otherwise the cost will be too high

![image](https://github.com/langgenius/dify/assets/17798310/0fe4a794-8142-4b77-b1d5-fa1084ece6b8)

**Comment by crazywoola** [comment]
You can switch your system reasoning model to gpt-4o.
